src/views/partials/email-breach-alert.js

```diff
@@ -0,0 +1,134 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The email templates are migrated from `hbs` and I tried to only extract the inline style to make the file more readable. We decided to not make any fundamental changes to the established code for the initial migration.

Reply:
Oh nce, I was going to leave a comment about how picky email clients are, but the styles are still actually inline - good job! (It's probably still a good idea to test in actual mail clients, e.g. new lines might still cause issues, but any issues are probably relatively easily solvable then.)

Reply:
That’s a good point! I we see any strange behavior we could easily strip the new lines.